Packs/Core/Playbooks/playbook-Suspicious_process_execution_by_scheduled_task_on_a_sensitive_server_README.md

```diff
@@ -0,0 +1,56 @@
+This playbook handles "Suspicious process execution by scheduled task on a sensitive server" alerts.
+
+Playbook Stages:
+
+Investigation:
+
+During the alert investigation, the playbook will perform the following:
+
+- Checks the suspicious process reputation.
+- Search for related XDR agent alerts to identify any malicious activity on the server.
```

Comment:
Searches for related XDR agent alerts that find any malicious activity on the server.